main.tex

```diff
@@ -39,7 +54,58 @@ \section{Introduction}
 
 The use of standard denominations in the resulting CoinJoin transaction obscures the relationship between individual inputs and outputs, making the origins of each output ambiguous. Unfortunately standard denominations also present limitations, such as requiring change outputs from these CoinJoin transactions, and limiting the usefulness of the privacy enhanced outputs for payments of arbitrary amounts.
 
-\section{Our Contribution}
+\subsection{Limitations of ZeroLink}
+In this work, we are aiming to improve ZeroLink, the most popular Chaumian CoinJoin implementation for Bitcoin. We identify several privacy shortcomings and inefficiencies of the current protocol as implemented by Wasabi.
+
+\subsubsection{Round denominations}
+
+
+Due to the nature of blind signatures, mixed outputs of ZeroLink CoinJoins are restricted to a fixed set of value denominations which are multiples of a base denomination\footnote{approximately $0.1$\bitcoinA{}}.
+
+This creates friction when sending or receiving arbitrary amounts of Bitcoin, as the fixed denomination generally creates change which is smaller, both when mixing and when spending mixed outputs.
```

Comment:
I don't understand what you want to say here... that when you have a anonset coin worth 0.1, and you make a payment with it, the change is below 0.1 and thus you cannot mix again without consolidation?

Reply:
yes, and also that when you have an arbitrary unmixed amount, in general you will get change which also must be consolidated if mixing

Reply:
```suggestion
This creates friction when sending or receiving arbitrary amounts of Bitcoin, as the fixed denomination generally creates change which is smaller, both when remixing and when remixing the change of a payment.
```

Reply:
that change seems more confusing to me, when you are mixing some arbitrary amount you often get change because of the base denomination, and separately after mixing if you spend mixed outputs you get change for the same reason. you have to remix that change but this jumps ahead to change from remixing that change